server/services/store/sqlstore/workspaces.go

```diff
@@ -144,3 +145,62 @@ func (s *SQLStore) GetWorkspaceCount() (int64, error) {
 	}
 	return count, nil
 }
+
+func (s *SQLStore) GetUserWorkspaces(userID string) ([]model.UserWorkspace, error) {
+	var query sq.SelectBuilder
+
+	if s.dbType == mysqlDBType {
+		query = s.getQueryBuilder().
+			Select("Channels.ID", "Channels.DisplayName", "COUNT(focalboard_blocks.id)").
+			From("focalboard_blocks").
+			Join("ChannelMembers ON focalboard_blocks.workspace_id = ChannelMembers.ChannelId").
+			Join("Channels ON ChannelMembers.ChannelId = Channels.Id").
+			Where("ChannelMembers.UserId = ?", userID).
+			Where("focalboard_blocks.type = 'board'").
+			Where("focalboard_blocks.fields LIKE '%\"isTemplate\":false%'").
```

Comment:
This is dangerous, somebody can introduce in the board description the text `"isTemplate":false` and this is going to swallow that. Personally I prefer to filter it after the query is returned. At least for MySQL.

Reply:
That's not what happens. If someone types `"isTemplate":false` in UI, it gets saved as `\"isTemplate\":false` (escaped double quotes)

Regular board-
<img width="247" alt="Screenshot 2021-08-23 at 6 33 39 PM" src="https://user-images.githubusercontent.com/18575143/130451942-eb8ec44b-097a-4e23-889e-d199549a312f.png">

Board with description set as `"isTemplate":false`-
<img width="311" alt="Screenshot 2021-08-23 at 6 34 28 PM" src="https://user-images.githubusercontent.com/18575143/130452076-8e603432-75ab-4c17-98b8-a0e5e8f47b5d.png">